R/impact_central.R

```diff
@@ -318,25 +318,25 @@ get_birth_cohort <- function(data) {
 #' modelled. This does not account for the future disease burden averted through
 #' current vaccine activities.
 #'
-#' @param baseline_impact Data frame of baseline impact data this needs to have
+#' @param baseline_burden Data frame of baseline impact data this needs to have
```

Comment:
Should we update the docs too?
```suggestion
#' @param baseline_burden Data frame of baseline burden data this needs to have
```

Reply:
You are right. We should update those. Will do after our meeting.